### PR TITLE
removed: [M3-7554, M3-7559] - Post Release Cleanup Parent/Child Feature Flags

### PR DESCRIPTION
**Pull request by jaalah-akamai**
## Description 📝
Remove the `parentChildAccountAccess` feature flag logic now that the feature has been in production for some time.

## Changes  🔄
- Removed feature flags from code, unit tests, and cypress e2e tests

## Target release date 🗓️
Next release: 5/28

### Verification steps
- Run unit tests: `yarn test`
- Run E2E tests:
```
yarn cy:run -s "cypress/e2e/core/account/account-cancellation.spec.ts,cypress/e2e/core/account/account-login-history.spec.ts,cypress/e2e/core/account/display-settings.spec.ts,cypress/e2e/core/account/personal-access-tokens.spec.ts,cypress/e2e/core/account/user-permissions.spec.ts,cypress/e2e/core/account/user-verification-banner.spec.ts,cypress/e2e/core/account/users-landing-page.spec.ts,cypress/e2e/core/billing/billing-contact.spec.ts,cypress/e2e/core/billing/restricted-user-billing.spec.ts,cypress/e2e/core/parentChild/account-switching.spec.ts,cypress/e2e/core/parentChild/token-expiration.spec.ts"
```
- Ensure that parent child features are not displayed for default accounts.
- Ensure that the features are enabled for parent/child accounts.

## As an Author I have considered 🤔

*Check all that apply*

- [X] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [ ] ➕ Adding a changeset
- [x] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

---
## Commit message and pull request title format standards

> **Note**: Remove this section before opening the pull request
**Make sure your PR title and commit message on squash and merge are as shown below**

`<commit type>: [JIRA-ticket-number] - <description>`

**Commit Types:**
- `feat`: New feature for the user (not a part of the code, or ci, ...).
- `fix`: Bugfix for the user (not a fix to build something, ...).
- `change`: Modifying an existing visual UI instance. Such as a component or a feature.
- `refactor`: Restructuring existing code without changing its external behavior or visual UI. Typically to improve readability, maintainability, and performance.
- `test`: New tests or changes to existing tests. Does not change the production code.
- `upcoming`: A new feature that is in progress, not visible to users yet, and usually behind a feature flag.

**Example:** `feat: [M3-1234] - Allow user to view their login history`

---

### CODE REVIEW

File: packages/manager/.changeset/pr-10489-removed-1716225501082.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@linode/manager": Removed
+---
+
+Post release cleanup of parent/child feature flags ([#10489](https://github.com/linode/manager/pull/10489))
```
Comment (mjac0bs) [inline packages/manager/.changeset/pr-10489-removed-1716225501082.md:5]:
```suggestion
`parentChildAccountAccess` feature flag ([#10489](https://github.com/linode/manager/pull/10489))
```

**Review by mjac0bs — APPROVED** [review]
Thanks for removing the flag. It is done! 🎉 

- Code looks good with removal of flag.
- Manually confirmed that parent (unrestricted and restricted), proxy, and child account basic flows to confirm users had expected enabled/disabled states and things like API tokens are being created correctly. Did not see any regressions.
- Confirmed default accounts don't see account switching UI components.
- Confirmed account switching is still working as expected.
- Confirmed the feature flag is no longer present in our dev tools.
- Confirmed relevant tests pass - the one failure in CI is an unrelated create-stackscripts test which was failing last week and fixed... this branch might not have been created off the latest `develop`.

**Comment by abailly-akamai** [comment]
@jaalah-akamai any reason why this contribution is not coming from your fork?

**Comment by jaalah-akamai** [comment]
See: https://github.com/linode/manager/pull/10497

**Pull request closed without merging**